### PR TITLE
Rebase filter fixes

### PR DESCRIPTION
**Pull request by dnil**
Same-same - but different.

### CODE REVIEW

File: puzzle/plugins/vcf/vcf.py [hunk 1]
```diff
@@ -79,6 +79,7 @@ def __init__(self, root_path=None, case_lines=None,
         self.filters.can_filter_impact_severity = True
         self.filters.can_filter_sv = True
         self.filters.can_filter_inheritance = True
+        self.filters.can_filter_sv_len=True
```
Comment (moonso) [inline puzzle/plugins/vcf/vcf.py]: Wounder why this was not there before...

File: puzzle/server/blueprints/variants/templates/macros/filters.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
           <input type="text" class="form-control" name="cadd" type="number" placeholder="23.4" step="0.1" value="{{ filters.cadd or '' }}">
         </div>
       {% endif %}
-      {% if db.filters.can_filter_sv_len %}
+      {% if db.filters.can_filter_sv_len and db.variant_type == 'sv'%}
```
Comment (moonso) [inline puzzle/server/blueprints/variants/templates/macros/filters.html]: :+1:

File: puzzle/server/blueprints/variants/templates/sv_variants.html [hunk 3]
```diff
@@ -42,10 +43,21 @@ <h4 class="panel-title">
             <td>{{ variant.cytoband_start or '-' }}</td>
             <td>{{ variant.cytoband_stop or '-' }}</td>
             <td align="right">{{ '{0:,}'.format(variant.sv_len) }}</td>
+	    <td>
```
Comment (moonso) [inline puzzle/server/blueprints/variants/templates/sv_variants.html]: What was this for?

Comment (moonso) [inline puzzle/server/blueprints/variants/templates/sv_variants.html]: Do you want to show frequencies for svs?

Comment (dnil) [inline puzzle/server/blueprints/variants/templates/sv_variants.html]: Exactly! :-) This is often implicit in score/rank, but helps in sifting through variants w/o clicking them.

**Comment by moonso** [comment]
Hello @dnil , I would really like to merge this but currently there seems to be some rebase problems and test failures. Could we sit together and look at it?

**Comment by dnil** [comment]
Thanks @moonso! I guess this is the second refactor since those fixes came about. :-D No worries, I’ll try to find a couple of minutes - shouldn’t be a problem.No major refactor planned for today? Otherwise I’ll wait.. ;)

// Daniel

**Comment by moonso** [comment]
There is one coming up actually....
but we could fix yours before if you like!

**Comment by dnil** [comment]
lol - I better get to it then!

**Comment by moonso** [comment]
:speedboat:

**Pull request closed without merging**